miner/worker.go

```diff
@@ -723,6 +743,7 @@ func (w *worker) makeCurrent(parent *types.Block, header *types.Header) error {
 		family:    mapset.NewSet(),
 		uncles:    mapset.NewSet(),
 		header:    header,
+		traceEnv:  traceEnv,
 	}
```

Comment:
I think it's not necessary, because `w.current.accRows` is assigned and maintained by `w.circuitCapacityChecker` 

(`accRows, err := w.circuitCapacityChecker.ApplyTransaction(traces)` & `w.current.accRows = accRows`)

resetting `w.circuitCapacityChecker` will be enough (I do that in `(w *worker) commitNewWork()`).